components/tag/stories/tag.test.js

```diff
@@ -1,8 +1,9 @@
 import { Variants } from "@spectrum-css/preview/decorators";
-import { Template } from "./template.js";
+import { TagsDefaultOptions, Template } from "./template.js";
 
 export const TagGroups = Variants({
 	Template,
+	SizeTemplate: TagsDefaultOptions,
```

Comment:
Realized after looking at Swatch yesterday that this is a much cleaner way to handle showing a group for sizing, so I made a few adjustments to the template, I also realized we probably don't need the selected template anymore so removed it.